admin/products_to_categories.php

```diff
@@ -351,19 +351,11 @@ function array_minus_array($a, $b) {
         break;
 
       case 'update_product':
-        $zv_check_master_categories_id = 'true';
         $new_categories_sort_array[] = $_POST['current_master_categories_id'];
         $current_master_categories_id = $_POST['current_master_categories_id'];

```

Comment:
Both of these POSTs should be (int) here too, because of the way they're used later. (Don't want them accidentally coming in as an array or string when going thru parsing loops, and there are some outputs later that are easier to clean up if `int`'d here.)